.github/workflows/release-prod.yaml

```diff
@@ -19,6 +19,24 @@ jobs:
         uses: actions/setup-go@v5
         with:
           go-version-file: go.mod
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to Registry
+        uses: docker/login-action@v2
+        with:
+          registry: registry.replicated.com
+          username: ${{ secrets.REGISTRY_USERNAME_PRODUCTION }}
+          password: ${{ secrets.REGISTRY_PASSWORD_PRODUCTION }}
+
+      - name: Build and push local-artifact-mirror image
+        run: |
+          export SHORT_SHA=dev-$(git rev-parse --short=7 HEAD)
```

Comment:
Are you sure that this should be using a short version and not the tag? 

Also, you should push the tag version to both prod and staging registries

Reply:
Also... I would kind of recommend putting this as a different job, both here and in the onpr/onmerge workflows

Reply:
That is a good question. I guess we need it published for all releases we create (CI, staging and production). I think @sgalsaleh can answer this.

Reply:
Yeah, let's use the tag when pushing to prod. We can push the tag to staging too, but IDK what would use it since I assume our CI will use the short SHA version.

Reply:
Done, please take a look.

Reply:
> Yeah, let's use the tag when pushing to prod. We can push the tag to staging too, but IDK what would use it since I assume our CI will use the short SHA version.

Any time that we run test apps with real release versions on staging